aws/resource_aws_api_gateway_integration_test.go

```diff
@@ -89,6 +89,28 @@ func TestAccAWSAPIGatewayIntegration_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_api_gateway_integration.test", "request_templates.application/xml", "#set($inputRoot = $input.path('$'))\n{ }"),
 				),
 			},
+
+			{
+				Config: testAccAWSAPIGatewayIntegrationConfigCacheKeyParameters,
```

Comment:
Do you mean adding a new function named `func TestAccAWSAPIGatewayIntegration_cache_key_parameters(t *testing.T)` (or similar) to the same file `resource_aws_api_gateway_integration_test.go`?